pages/docs/adapters/calculix/adapter-calculix-configure.md

```diff
@@ -154,6 +177,10 @@ must be changed to
 
 Note that an error will only occur if nodes-mesh-with-connectivity is specified without a .sur file. The calculix-adapter with nearest-projection mapping only supports tetrahedral elements (C3D4 and C3D10) as preCICE only works with surface triangles for nearest-projection mapping.
 
+### Modal dynamic simulations
+
+The adapter supports modal dynamic simulations, where eigenmodes from a frequency analysis are used. Run the frequency analysis without preCICE (`ccx_preCICE -i [CalculiX input file]`) then run a coupled simulation. Currently, only explicit coupling is supported.
```

Comment:
Half a sentence more explaining the concept behind modal dynamic simulations would be appreciated.